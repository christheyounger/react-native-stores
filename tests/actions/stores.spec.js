import {
  getStores,
  gotStores,
  getStore,
  gotStore
} from '../../app/actions/stores';

describe('actions.stores', () => {
  test('should generate GET_STORES', () => {
    expect(getStores()).toMatchSnapshot();
  });
  test('should generate GOT_STOERS', () => {
    expect(gotStores([{name: 'store 1'}])).toMatchSnapshot();
  });
  test('should generate GET_STORE', () => {
    expect(getStore()).toMatchSnapshot();
  });
  test('should generate GOT_STORE', () => {
    expect(gotStore({name: 'store 1'})).toMatchSnapshot();
  });
});
