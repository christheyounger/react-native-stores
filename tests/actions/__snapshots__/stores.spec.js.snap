// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions.stores should generate GET_STORE 1`] = `
Object {
  "type": "GET_STORE",
}
`;

exports[`actions.stores should generate GET_STORES 1`] = `
Object {
  "type": "GET_STORES",
}
`;

exports[`actions.stores should generate GOT_STOERS 1`] = `
Object {
  "payload": Object {
    "stores": Array [
      Object {
        "name": "store 1",
      },
    ],
  },
  "type": "GOT_STORES",
}
`;

exports[`actions.stores should generate GOT_STORE 1`] = `
Object {
  "payload": Object {
    "store": Object {
      "name": "store 1",
    },
  },
  "type": "GOT_STORE",
}
`;
